src/scss/index.scss

```diff
@@ -1 +1,305 @@
+$primary: #26BFBF;
+$secondary: #F39237;
+@import "../../node_modules/bootstrap/scss/bootstrap";
 
+// Path: src/scss/styles.scss
+
+$colours: (
+  
+  "secondary": #F39237,
+  "tertiary": #F7B678b8, 
+  "quaternary": #26bfbf8b,
+  "white-text": #ffffff,
+  "black-text": #000000,
+  "invalide": #ff0000,
+);
+
+@mixin button-border($radius) {
+    border-radius: $radius;
+    border: 2px solid $primary;
+    color: $primary;
+}
+
+@mixin flex-form {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: stretch;
+  // margin-bottom: 2rem;
+  font-size: 2rem;
+}
+@mixin main-border {
+  border: 15px solid $primary;
+  
+}
+
+@mixin invalid-feedback {
+  color: map-get($colours, "invalide");
+  display: none;
+}
+
+@mixin input-border($radius) {
+  border-radius: $radius;
+  border: 2px solid $primary;
+}
+
+@mixin vibe-color {
+  color: map-get($colours, "secondary");
+  font-family: 'Sail', cursive;
+  font-size: 2rem;
+}
+
+@mixin connect-color {
+  color: $primary;
+  font-family: 'Sail', cursive;
+  font-size: 2rem;
+};
+@mixin post-img {
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+  object-fit: cover;
+  border: 2px solid map-get($colours, "secondary");
+  margin: 0 0 -3rem 0;
+};
+@mixin reply-img {
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+  object-fit: cover;
+  border: 2px solid $primary;
+  margin: 0 0 -3rem 0;
+};
+@mixin reply-container {
+  margin: 1rem 2rem;
+  background-color: map-get($colours,"quaternary" );
+}
+.nav-bottom-border {
+  border-bottom: 2px solid map-get($colours, "secondary");
+  margin-bottom: 1rem;
+  padding-bottom: 1rem;
+};
+body {
+  @include main-border;
+  border-bottom: none;
+  font-family: poppins;
+  height: 200vh;
+  margin-bottom: 0rem;
+  
+  
+};
+nav {
+  @include main-border;
+  border-bottom: none;
+  margin-bottom: 0rem;
+  height: 8rem;
+};
+.footer-border {
+  border-right: 15px solid $primary;
+  border-left: 15px solid $primary;
+  border-bottom: 15px solid $primary;
+  
+  
+}
+.login-page{
```

Comment:
Use bootstrap class instead

Reply:
I just forgot to delete it. Wasnt in use :)